src/js/tracing/utils.ts

```diff
@@ -0,0 +1,34 @@
+import { IdleTransaction, SpanStatus } from "@sentry/tracing";
+
+/**
+ * Converts from milliseconds to seconds
+ * @param time time in ms
+ */
+export function msToSec(time: number): number {
+  return time / 1000;
+}
```

Comment:
We don't use this function, or do we?

Reply:
No we don't, it was copied over from js at first, let me remove it.